src/main/web/templates/handlebars/list/partials/paginator.handlebars

```diff
@@ -10,7 +10,7 @@
             {{#each paginator.pages}}
             <li class="margin-top-sm--0 margin-top-md--0">
                 {{#if_eq this paginator.currentPage}}
-                <a href="" class="page-link btn btn--plain btn--plain-active btn--block" aria-current="true" aria-label="Page {{this}} of {{paginator.numberOfPages}}">{{this}}</a>
+                <a href="" class="page-link btn btn--plain btn--plain-active btn--block" aria-current="true" aria-label="Current page {{this}} of {{paginator.numberOfPages}}">{{this}}</a>
```

Comment:
I think the brackets and capital P are required to help playback in voiceover as it will read it back as:
> "Current page."
> "Page n of x."

`Current page (Page {{this}} of {{paginator.numberOfPages}}">{{this}})`